css/bootstrap.css

```diff
@@ -6758,3 +6771,43 @@ button.close {
   }
 }
 /*# sourceMappingURL=bootstrap.css.map */
+
+
+footer
```

Comment:
Similar to how Chris split up his parts, you should create a new folder for header and footer. And then add the css in it. So something like:
/header/frontend/css/main.css
/footer/frontend/css/main.css